observability/WFLY-20296-add-support-for-more-opentelemetry-config-options.adoc

```diff
@@ -0,0 +1,154 @@
+---
+categories:
+- observability
+stability-level: community
+# Specify the Feature Development tracker issue for the feature.
+# This must be an issue tracked in https://github.com/orgs/wildfly/projects/7/views/1.
+# To create a Feature Development tracker issue, go to  https://github.com/wildfly/wildfly-proposals/issues/new/choose
+# and select 'Feature Development'
+issue: https://github.com/wildfly/wildfly-proposals/issues/683
+feature-team:
+ developer: jasondlee
+ sme:
+  -
+ outside-perspective:
+  - TomasHofman
+promotes:
+promoted-by:
+---
+= WFLY-20296 - Add support for more OpenTelemetry configuration options
+:author:            Jason Lee
+:email:             jasondlee@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+While every aspect of OpenTelemetry is technically configurable via MP Telemetry and MP Config, there have been several requests for more direct support for the various options in the management model. The RFE will identify and implement the options most likely to be of general interest to WildFly users.
+
+=== User Stories
+
+- As an application developer, I would like greater control in how OpenTelemetry is configured in the server without having to depend on an external library (e.g., MicroProfile Config).
```

Comment:
Out of curiosity, it's not clear to me how exactly it affects the developer to rely on MP Config to configure opentelemetry.

My understanding is that the MP Config way simply means that I would e.g. define a system property via `-Dotel.service.name=myname` (tried and it works), or I could create "permanent" system property in standalone.xml. Perhaps I can also package an application.properties file into my deployment (haven't tried that).

Nothing else is needed on the application side (no special dependency, etc). Am I correct? The "external library" is a detail not visible to the app developer.

Reply:
I more had in mind the server administrator, as the MP Config extension/subsystem would have to be enabled to make it work. I'll try to clean up that wording there.

Reply:
OK, I get what you meant now.

Reply:
+1 to focusing on an administrator.

Beyond not 'depending on a library' (or setting system properties) in terms of a programming API, someone using this is *depending on us* for a robust, long-term-compatible API. They are depending on us to insulate them from underlying changes in OTel. If OTel renames one of their props, or whatever, in a future release, that is our problem, not the user's problem. They count on us to shield them from such things. That is what the WildFly management API does for people who use the many libraries we integrate.

So, think very hard about what you expose, as you'll be living with it for a long time. :-)